src/app/authentication/sagas.ts

```diff
@@ -4,10 +4,17 @@ import { call, put, takeLatest } from 'redux-saga/effects';
 import { client } from '../../graphql/api';
 import { Logger } from '../utils/logger';
 
-import { LOGIN, REGISTER } from './actions';
+import { LOGIN, LOGOUT, REGISTER } from './actions';
+
+export function* logoutSaga(): SagaIterator {
+    yield call(client.logout);
+    yield put(LOGOUT.COMPLETED());
+}
 
 export function* registerSaga(action: ReturnType<typeof REGISTER.TRIGGER>): SagaIterator {
     yield call(client.register, action.payload);
```

Comment:
Why you calling register twice? Also why you don't have .failed handling, example in loginSaga